### PR TITLE
adds alternative anime search in context menu

### PR DESCRIPTION
**Pull request by SwagOtaku**
helpful when anime movies/ovas don't work or having problems playing episode sources

example: https://streamable.com/m01p9

 thanks to gogoanime and mp4upload code from github user vn-ki (anime-downloader)

### CODE REVIEW

**Comment by SwagOtaku** [comment]
*when playing anime through search alt, no watchlist update

**Comment by icpryde** [comment]
@SwagOtaku This is an awesome addition with alt search, just noticed when pulling information up can you make the lists it populates follow the same rules for viewtypes like you just added in your other PR. (tv shows, episodes, seasons). Right now if you use alt it will just make all view types "tv shows".

**Comment by phresh-prince** [comment]
I'm new to GitHub but have been using Kodi for a few years. I can't seem to get this to work. I have wonderfulsubs v0.0.19 but no alt search in the menu when playing an episode. Am I missing something or is this an upgrade to the standard add-on? Please help, thanks.

**Comment by phresh-prince** [comment]
SwagOtaku, any help/guide to use or install this would be greatly appreciated. Can you please help? Thanks.

**Comment by icpryde** [comment]
@phresh-prince heres a tutorial to do what you are asking. https://gist.github.com/Chaser324/ce0505fbed06b947d962

**Comment by SwagOtaku** [comment]
@phresh-prince if you want to test this pull request, download this zip https://github.com/SwagOtaku/plugin.video.wonderfulsubs-1.git

File: resources/lib/WonderfulSubsBrowser.py [hunk 1]
```diff
@@ -62,14 +64,23 @@ def _parse_filtered_anime_view(self, res, base):
...
+                                              base["plot"],
+                                              self._draw_cm))
 
         return result
 
     def _parse_history_view(self, res):
         name = res
         return utils.allocate_item(name, "search/" + name + "/1", True)
 
+    def _draw_cm(self, sysaddon, name):
```
Comment (DxCx) [inline resources/lib/WonderfulSubsBrowser.py]: this doesnt belong to WonderfulSubs browser
it should be part of default.py and passed as callback to the others.

**Comment by SwagOtaku** [comment]
ok will do

**Comment by icpryde** [comment]
When this gets added can you add the ability to use alt-search at the episode list as well. Currently you can only enable it at the tv shows/tvshow poster section.

**Comment by SwagOtaku** [comment]
@DxCx made the requested changes

**Comment by SwagOtaku** [comment]
@icpryde can't do that since the context menu uses the title of the selected item, so if search-alt were added to episode page it would search for something `Ep. 10 ...` with no results.

 But added the ability to use search alt at watchlists page

**Comment by icpryde** [comment]
Oh ok that’s still awesome thank you @SwagOtaku

**Pull request merged**